src/league.js

```diff
@@ -460,11 +460,7 @@ league_attributes = {
     'getCaptains':function() {
         var self = this;
         return Q.fcall(function() {
-            var captains = [];
-            _.each(self._teams, function(team) {
-                captains.push(team.captain);
-            });
-            return captains;
+            return _.map(self._teams, "captain");
```

Comment:
so this syntax is a little weird to me.

I expect `map` to apply a function to every `team` in `_teams`. 

How should I be reading this?

Clearly it returns an array of captains from `_teams` but it doesnt do this by applying a function - atleast not at this level of abstraction.